server/api.go

```diff
@@ -86,8 +87,19 @@ func (h *Handler) hasPermissionToChannel(userID, channelID string) (*model.Chann
 	return nil, false
 }
 
+var exportActive int32
```

Comment:
Is there a reason for using atomic around and not the cluster Mutex Mattermost exposes? Could this allow multiple exports to run at the same time if the request lands on a different plugin process?
([docs](https://github.com/mattermost/mattermost/blob/server/public/v0.1.4/server/public/pluginapi/cluster/mutex.go#L55))

Reply:
Yes, great point.  Can you point me to docs/code for the cluster mutex?

Reply:
Documentation in godoc: https://pkg.go.dev/github.com/mattermost/mattermost/server/public/pluginapi/cluster#NewMutex
Example in calls: [mattermost/mattermost-plugin-calls/server/sync.go](https://github.com/mattermost/mattermost-plugin-calls/blob/37bc6c6b6f85923e15dfd38950a1f4d74fd728d5/server/sync.go#L24)